tests/Tests.elm

```diff
@@ -0,0 +1,63 @@
+module Tests exposing (..)
+
+import Test exposing (..)
+import Expect
+
+--Do not delete these imports as they are beeing compiled even when not beeing tested directly
```

Comment:
Thougth so myself, but it failed afterwards - Almost same name on definition and import.